point center
point column_1
point column_2

